stockflux-watchlist/src/components/watchlist-card/WatchlistCard.js

```diff
@@ -26,7 +26,7 @@ const WatchlistCard = props => {
 
   useEffect(() => {
     StockFlux.getMiniChartData(props.symbol).then(response => {
-      if (response.success) {
+      if (response.data) {
```

Comment:
I don't understand how it worked for me when I checked for response.success, as it clearly should not have ever worked. Maybe I accidentally did not commit a line or something. Anyway, all works now. :)